src/app/services/work-item.service.ts

```diff
@@ -247,7 +247,7 @@ export class WorkItemService {
    */
   getWorkItemById(id: string): Observable<WorkItem> {
     if (this._currentSpace) {
-      return this.http.get(this._currentSpace.links.self + '/workitems/' + id)
+      return this.http.get(this._currentSpace.links.self.split('/spaces/')[0] + '/workitems/' + id)
```

Comment:
When do you need access to a WorkItem by ID where you don't have a WorkItem.link.self?